doc: Add "Managing Workspaces" Guide

zihanKuang opened the pull request:
**Notes for Reviewers**

This PR fixes #610

- This PR adds the `managing-workspaces.md` guide. It documents management operations for Workspaces, including creation, editing, deletion, and resource assignment.
- The "Deletion Behavior" section is moved from `workspaces.md` into this new guide.
- Screenshots and GIFs are added to illustrate the steps for each action.

leecalcote commented on content/en/cloud/spaces/managing-workspaces.md:
```diff
@@ -0,0 +1,167 @@
+
+The [Workspaces page](https://cloud.layer5.io/spaces/workspaces) is where you can see all of the workspaces within the currently selected organization.
+
+To suit different workflows, you can switch between two distinct layouts: a visual grid view and a detailed table view.
```
I wonder if this might be presented as a tip-style alert. Maybe so or maybe it would be unnecessarily distracting. Hmm.

leecalcote replied:
Also, I wonder if the "grid view" and "table view" words should be hyperlinked to their respective sections.

leecalcote commented on content/en/cloud/spaces/managing-workspaces.md:
```diff
@@ -0,0 +1,167 @@
+
+![Animation showing team assignment process](/cloud/spaces/images/assign-teams.gif)
+
+You can open the team management modal from either the grid or table view.
```
I wonder if we should be using the term "modal" in our user-facing docs.

leecalcote commented on content/en/cloud/spaces/managing-workspaces.md:
```diff
@@ -0,0 +1,167 @@
+3. Click **Save** to apply your changes.
+
+{{< alert type="info" title="Team and Workspace Relationships" >}}
+You can assign multiple teams to a single workspace, and a single team can also be a member of multiple workspaces. This provides flexible access control across your projects. For more restrictions, see [Workspaces documentation](https://docs.layer5.io/cloud/spaces/workspaces/).
```
Perhaps, the relative path?

leecalcote commented on content/en/cloud/spaces/workspaces.md:
```diff
@@ -92,6 +78,4 @@ To make the most of Meshery Workspaces, here are some best practices:
-
-[^1]: This functionality is not fully implemented yet. Users might occasionally observe that designs and views are preserved after Workspace deletion.
\ No newline at end of file
+Meshery Workspaces enhance collaboration within your teams, providing a structured environment for sharing and managing resources. By following best practices and understanding the core components of Workspaces, you can maximize the benefits of this feature in Meshery.
```
Does this page have a link to the "Managing Workspaces" page? Should it or are the Previous and Next page buttons sufficient in this case?

leecalcote commented:
Topics and questions to address either in these two pages or other pages:

1. When a design is transferred from one Workspace to the next, is design ownership affected? Who has access? What if the Workspaces are in different orgs? 
2. What other users of a Workspace do with my design or view? Delete my design? 
3. Can Workspaces contain private, public, and published designs? 
4. What are best practices of organizing my workspaces? One per team? One per project? One for private designs only? A separate workspace for published designs (designs that are intended to be reusable patterns)?

5. How will I know if another user made changes to my design or view? Will I be notified? If there an edit history or audit trail?
6. If I share my private design with a user in a different org, can they share my design with other users?
7.  My design is public and other users can edit my design, because of this. Is there a way to have my design publicly available, but read-only to other users?
8. When I import from a k8s manifest, helm chart or other type, a new design file is created. After making edits to my design, I can export my design in the same format from which I imported. Can I also download a copy of my original, unaltered file?
9. When I import from a k8s manifest, helm chart or other type, and choose to merge this file into an existing design, can I download my original file(s)?

@zihanKuang @vishalvivekm, please open issue(s) if these questions / topics are to be covered elsewhere.

leecalcote commented:
@nitish-bot, you might like to undertake #8 above. By "undertake", I mean document your empirical findings as you explore this feature.

leecalcote commented:
@zihanKuang are you merging?

zihanKuang commented:
@leecalcote, I've implemented the 'danger' styling to mark elements in red.
Regarding the question about workspace space or file count limitations, I've added it to the issue #616. We'll address this and other frequently asked questions in a QA section.

leecalcote commented:
Excellent. Yes, again, please merge.
